support.md

```diff
@@ -24,4 +25,5 @@ issues important to you in our [issue tracker][issues]:
 [issues]: https://github.com/flutter/flutter/issues
 [so]: https://stackoverflow.com/tags/flutter
 [mailinglist]: https://groups.google.com/d/forum/flutter-dev
-[gitter]: https://gitter.im/flutter/flutter
\ No newline at end of file
+[gitter]: https://gitter.im/flutter/flutter
+[reddit:]: https://www.reddit.com/r/FlutterDev/
```

Comment:
The link doesn't render properly because of the extra colon in the link name (`reddit:` instead of `reddit`):

> <img width="539" alt="screen shot 2018-08-06 at 06 40 02" src="https://user-images.githubusercontent.com/4140793/43712303-a84e37ce-9943-11e8-983d-83876b253479.png">